src/api/app/controllers/webui/request_controller.rb

```diff
@@ -285,6 +272,14 @@ def main_object
 
   private
 
+  def any_project_maintained_by_current_user?(actions)
+    # search for a project, where the user is not a package maintainer but a project maintainer and show
+    # a hint if that package has some package maintainers (issue#1970)
+    projects = actions.map { |action| action[:tprj] }.uniq
```

Comment:
We could replace this with `bs_request.bs_request_actions.select(:target_project).distinct.pluck(:target_project)`. That should be faster and we get rid of the dependency on the webui_actions hash

Reply:
we need many other fields, just look into the related helpers and other methods in the controller..